resources/public/api/conf/common/testing.md

```diff
@@ -0,0 +1,7 @@
+You can use the sandbox environment to [test this API](https://developer.service.hmrc.gov.uk/api-documentation/docs/testing). You can use the <link:Create Test User API> or its frontend service to create test users.
```

Comment:
is `<link:Create Test User API>` a placeholder?

Reply:
Yes. I should be able to add or delete once the testing requirements are agreed.